new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Helm.java

```diff
@@ -149,14 +160,43 @@ public static boolean list(HelmParams params) {
     return exec(String.format("helm list -n %s", params.getNamespace()));
   }
 
+  public static boolean getExpectedValues(HelmParams params, String expectedValue) {
```

Comment:
add javadoc

Reply:
What does this method do?

Reply:
changed this method to a more generic one in TestActions.java